adapters/src/main/java/com/squareup/moshi/adapters/EnumJsonAdapter.java

```diff
@@ -20,13 +20,23 @@
 import com.squareup.moshi.JsonDataException;
 import com.squareup.moshi.JsonReader;
 import com.squareup.moshi.JsonWriter;
+import com.squareup.moshi.Moshi;
 import java.io.IOException;
 import java.util.Arrays;
 import javax.annotation.Nullable;
 
 /**
  * A JsonAdapter for enums that allows having a fallback enum value when a deserialized string does
- * not match any enum value.
+ * not match any enum value. To use, add this as an adapter for your enum type on your {@link
+ * Moshi.Builder}:
+ *
+ * <pre> {@code
```

Comment:
I usually always see and do `<pre><code>`, is there a difference?

Reply:
`<code>` doesn’t escape `<` and `&` characters.

Reply:
(in this case it’s the same, but `{@code}` is a better default.)

Reply:
:+1: